ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/fonts/TypographyType.kt

```diff
@@ -0,0 +1,22 @@
+package com.revenuecat.purchases.ui.revenuecatui.fonts
+
+/**
+ * Text styles for which the fonts can be overridden in the paywall.
+ */
+enum class TypographyType {
```

Comment:
Android doesn't expose the text styles in the typography, so we are forced to mimick them with a new enum.